woocommerce/woocommerce-ios
# [IAP] Updated design for Free Trial summary cards

Partially closes: #10192 

## Description
This PR addresses updating the UI for the current summary banner/card for the cases of "Free Trial" and "Free Trial expired", that is, the merchant has a Free Trial site, and visits the Upgrades view in order to potentially upgrade to a paid plan.

This PR only addresses 2 of the 4 cases described in the issue linked above. It does not cover:
- Summary banner/card hiding on scroll-down
- Active subscription case: Since at the moment merchants can only upgrade from Free Trial to Paid plan, and any cross-grade is done directly via the App Store, they'll never see this view. We will be able to address this case in future milestones when cross-grading is allowed in-app.

Note as well that the green/red-coloured image was removed from the designs ( p1690534419475089?thread_ts=1688979147.117499&cid=C0354HSNUJH-C0354HSNUJH )

## Testing instructions
1. On a Free Trial site, tap on Upgrade Now
2. See how the top of the screen shows the updated design, with the "You're in a Free Trial" message.
3. Change the expiration date to less than zero days, for example retuning -1 on `daysLeft()` [here](https://github.com/woocommerce/woocommerce-ios/blob/5c65a01e38f4b1a001abb9906520c80e219a8537/WooCommerce/Classes/ViewRelated/Upgrades/SubscriptionsViewModel.swift#L247)
4. See how the top of the screen shows the updated design, with the "Your Free Trial has ended" message


## Screenshots
<img width="754" alt="010823_updated_summary_banner" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/bc5b9071-360a-443f-b629-25065e14bf95">
